packages/react-storage/src/styles/storage-browser.css

```diff
@@ -1,5 +1,8 @@
 .storage-browser {
-  --storage-browser-gap: 0.6rem;
+  --storage-browser-action-menu-width: 220px;
```

Comment:
Worry a bit with hardcoded values. Can we just constrain the menu width by the width of its content?

Reply:
Yeah, updated the width to be 'max-content' and that seems to work fine. (screenshot of it open)

<img width="175" alt="Screenshot 2024-07-24 at 12 40 39 PM" src="https://github.com/user-attachments/assets/28d8771a-947f-42ed-b212-31812eb126c0">